frontend/src/assets/styles/global.scss

```diff
@@ -1490,9 +1490,18 @@ cds-table-header-cell {
   background: var(--light-theme-layer-accent-layer-accent-01, #dfdfe1);
 }
 
-:is(cds-table-header-cell, cds-table-cell):is(:first-child, :last-child) {
-  padding: 0;
-  width: 1.5rem;
+#submissions-table {
```

Comment:
We shouldn't be doing exclusive classes for any table we add in the application. Please keep things in a generic way.

Reply:
That's because the submissions table has a "padding" to the right of the table that the search table does not have.
Both have the "padding" to the left, but only the first one has the "padding" to the right.

Reply:
But I can do it in a more generic way.

Reply:
done

Reply:
And just to clarify, what I'm calling "paddings" are actually empty columns with a specific style.
So, now I moved this styles to a new class and applied it to the specific columns.

Reply:
And it's ready to review again.